play.pokemonshowdown.com/src/panel-teambuilder.tsx

```diff
@@ -168,6 +168,7 @@ class TeambuilderPanel extends PSRoomPanel<TeambuilderRoom> {
 		// before: Dragged goes before Over, splice at i
 		// after: Dragged goes after Over, splice at i - 1 + 1
 		PS.teams.list.splice(iOver, 0, draggedTeam);
+		PS.teams.save();
```

Comment:
Hm. This is saving too often, I think.

Reply:
Like this is happening every team you drag past. Feels excessive to save/load based on that...